airbyte-integrations/connectors/destination-redshift/src/main/resources/spec.json

```diff
@@ -23,48 +23,48 @@
         "maximum": 65536,
         "default": 5439,
         "examples": ["5439"],
-        "title": "Port"
+        "title": "Port *"
       },
       "username": {
         "description": "Username to use to access the database.",
         "type": "string",
-        "title": "Username"
+        "title": "Username *"
       },
       "password": {
         "description": "Password associated with the username.",
         "type": "string",
         "airbyte_secret": true,
-        "title": "Password"
+        "title": "Password *"
       },
       "database": {
         "description": "Name of the database.",
         "type": "string",
-        "title": "Database"
+        "title": "Database *"
       },
       "schema": {
         "description": "The default schema tables are written to if the source does not specify a namespace. Unless specifically configured, the usual value for this field is \"public\".",
         "type": "string",
         "examples": ["public"],
         "default": "public",
-        "title": "Default Schema"
+        "title": "Default Schema *"
       },
       "s3_bucket_name": {
-        "title": "S3 Bucket Name",
+        "title": "S3 Bucket Name (Optional)",
         "type": "string",
         "description": "The name of the staging S3 bucket to use if utilising a COPY strategy. COPY is recommended for production workloads for better speed and scalability. See <a href=\"https://docs.aws.amazon.com/redshift/latest/dg/c_loading-data-best-practices.html\">AWS docs</a> for more details.",
         "examples": ["airbyte.staging"]
       },
       "s3_bucket_path": {
-        "title": "S3 Bucket Path",
+        "title": "S3 Bucket Path (Optional)",
         "type": "string",
         "description": "The directory under the S3 bucket where data will be written. If not provided, then defaults to the root directory.",
         "examples": ["data_sync/test"]
       },
       "s3_bucket_region": {
-        "title": "S3 Bucket Region",
+        "title": "S3 Bucket Region (Optional)",
         "type": "string",
         "default": "",
-        "description": "The region of the S3 staging bucket to use if utilising a copy strategy.",
+        "description": "The region of the S3 staging bucket to use if utilising a COPY strategy. See the docs for details. See the docs for details. https://docs.aws.amazon.com/AmazonS3/latest/userguide/creating-bucket.html#:~:text=In-,Region,-%2C%20choose%20the%20AWS",
```

Comment:
Why is the URL not hyperlinked? And I think you've put the text in twice for 'see the docs..'?